test/new-e2e/tests/windows/common/powershell/command_builder.go

```diff
@@ -150,6 +150,33 @@ func (ps *powerShellCommandBuilder) WaitForServiceStatus(serviceName, status str
 	return ps
 }
 
+// DisableWindowsDefender creates a command to try and disable Windows Defender without uninstalling it
+func (ps *powerShellCommandBuilder) DisableWindowsDefender() *powerShellCommandBuilder {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
func (ps *powerShellCommandBuilder) WindowsDefenderCommand() *powerShellCommandBuilder {
```